packages/core/src/NavigationContainer.tsx

```diff
@@ -134,6 +137,10 @@ const Container = React.forwardRef(function NavigationContainer(
     [trackAction]
   );
 
+  const getRootState = () => {
+    return Object.values(stateGetters.current)[0]();
```

Comment:
Instead of getting the first value, can we be more explicit and get `stateGetters.current.root()`?